lib/client/backend_client.dart

```diff
@@ -20,32 +25,56 @@ class BackendClient {
     } catch (e) {
       print('Error during HTTPrequest $e');
       return 'Error during HTTP request: $e';
-    }
+    } // } finally {
+    //   // JACKSON: The finally block was added due to adding the httpClient rather than the one-off requests
+    //   httpClient.close();
+    // }
```

Comment:
Please remove this since it is commented out code.

Reply:
Done!